File: data/python/COUNT_NUMBER_OF_WAYS_TO_COVER_A_DISTANCE_1_MAIN.py
if __name__ == '__main__':
    param = [
    (37,),
    (82,),
    (87,),
    (80,),
    (92,),
    (58,),
    (98,),
    (53,),
    (11,),
    (58,)
        ]
    n_success = 0
    print("class,mutant,test_index,result")
    for i, parameters_set in enumerate(param):
        try:
            print("COUNT_NUMBER_OF_WAYS_TO_COVER_A_DISTANCE_1," + sys.argv[1] + "," + str(i) + "," + str(f_gold(*parameters_set)))
        except:
            print("COUNT_NUMBER_OF_WAYS_TO_COVER_A_DISTANCE_1," + sys.argv[1] + "," + str(i) + ",EXCEPTION")

File: data/java/COUNT_NUMBER_OF_WAYS_TO_COVER_A_DISTANCE_1_MAIN.java
public static void main(String args[]) throws IOException {
    int n_success = 0;
    List<Integer> param0 = new ArrayList<>();
    param0.add(37);
    param0.add(82);
    param0.add(87);
    param0.add(80);
    param0.add(92);
    param0.add(58);
    param0.add(98);
    param0.add(53);
    param0.add(11);
    param0.add(58);
    StringBuilder builder = new StringBuilder();
    builder.append("class,mutant,test_index,result");
    FileWriter writer = new FileWriter(args[0]);
    for(int i = 0; i < param0.size(); ++i)
    {
        try {
	        if(f_filled(param0.get(i)) == f_gold(param0.get(i)))
	        {
                builder.append("COUNT_NUMBER_OF_WAYS_TO_COVER_A_DISTANCE_1," + args[1] + "," + i + ",SUCCESS\n");
            } else {
                builder.append("COUNT_NUMBER_OF_WAYS_TO_COVER_A_DISTANCE_1," + args[1] + "," + i + ",FAILURE\n");
            }
        } catch (Exception e) {
            builder.append("COUNT_NUMBER_OF_WAYS_TO_COVER_A_DISTANCE_1," + args[1] + "," + i + ",EXCEPTION\n");
        }
    }
    writer.write(builder.toString());
    writer.close();
}
